packages/idx/src/property/search.component.html

```diff
@@ -52,7 +52,7 @@ <h2 data-ng-show="widgetName" data-ng-bind="widgetName"></h2>
                  data-ng-if="(options.query.where.City.length || options.query.where.CountyOrParish.length || options.query.where.MLSAreaMajor.length || options.query.where.PostalCode.length)">
                 <span data-ng-if="options.query.where.City.length" ng-bind="options.query.where.City"></span>
                 <!-- NOTE: these are actually arrays so we union them then join split with a comma -->
-                <span data-ng-if="options.query.where.CountyOrParish.length || options.query.where.MLSAreaMajor.length || options.query.where.PostalCode" ng-bind="(options.query.where.City.length && (options.query.where.CountyOrParish.length || options.query.where.MLSAreaMajor.length || options.query.where.PostalCode.length) ? ', ' : '') + _.join(_.union(options.query.where.CountyOrParish, options.query.where.MLSAreaMajor, options.query.where.PostalCode), ', ')"></span>
+                <span data-ng-if="options.query.where.CountyOrParish.length || ouptions.query.where.MLSAreaMajor.length || options.query.where.PostalCode" ng-bind="(options.query.where.City.length && (options.query.where.CountyOrParish.length || options.query.where.MLSAreaMajor.length || options.query.where.PostalCode.length) ? ', ' : '') + _.join(_.union(options.query.where.CountyOrParish, options.query.where.MLSAreaMajor, options.query.where.PostalCode), ', ')"></span>
```

Comment:
There's a misspelling of options as `ouptions`.

Reply:
I'll fix that along with other additions

Reply:
fixed with https://github.com/Sitetheory/stratus/pull/526